adk/ADK.py

```diff
@@ -91,8 +94,18 @@ def write_to_pipe(self, payload, pprint=print):
     def process_local(self, local_payload, pprint):
         result = self.apply(local_payload)
         self.write_to_pipe(result, pprint=pprint)
+        
+    def mlops_init(self):
+        mlops_token = os.environ.get("DATAROBOT_MLOPS_API_TOKEN", None)
```

Comment:
Is there a reason you aren't using the same env var as the monitoring agent itself? This might make things simpler, unless you specifically don't want them to be the same. That would be just `MLOPS_API_TOKEN`.

Reply:
Yeah, in the event that there are other MLOPS API tokens from different orgs; wanting to be very clear about what this token is used for so it's not confused with any algorithmia token